products/sle/main.pm

```diff
@@ -937,6 +939,15 @@ sub prepare_target() {
     }
 }
 
+sub load_rollback_tests() {
+    loadtest "boot/grub_test_snapshot";
+    if (get_var('UPGRADE') || get_var('ZDUP')) {
+        loadtest "boot/snapper_rollback";
+    }
+    if (get_var('MIGRATION_ROLLBACK')) {
+        loadtest "online_migration/sle12_online_migration/snapper_rollback";
+    }
+
```

Comment:
while you are at it. It would be great if you can crosscheck with the opensuse case. Especially update/upgrade on openSUSE Leap is getting more and more important ( @lnussel )